plugins/mock/uca-mock-camera.c

```diff
@@ -375,6 +375,43 @@ uca_mock_camera_grab (UcaCamera *camera, gpointer data, GError **error)
     return TRUE;
 }
 
+static gboolean
+uca_mock_camera_grab_live (UcaCamera *camera, gpointer data, GError **error)
```

Comment:
Can you make one function which is called from both, the normal `grab` and `grab_live` instead of duplicating code?